delocate/tests/test_delocating.py

```diff
@@ -14,7 +14,7 @@
 from ..delocating import (DelocationError, delocate_tree_libs, copy_recurse,
                           delocate_path, check_archs, bads_report,
                           filter_system_libs)
-from ..libsana import (tree_libs, search_environment_for_lib)
+from ..libsana import (tree_libs_from_directory, search_environment_for_lib)
```

Comment:
Correct me if I am wrong - but I think we are no longer testing `tree_libs`.  I know we've deprecated it, but it would be good to keep the tests until we have removed it.

Reply:
Done, I've now parametrized the test.  `test_delocate_tree_libs` now tests both functions with the same test.